src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsCheckBoxUI.java

```diff
@@ -39,7 +39,7 @@
  */
 public class WindowsCheckBoxUI extends WindowsRadioButtonUI
 {
-    // NOTE: MetalCheckBoxUI inherts from MetalRadioButtonUI instead
+    // NOTE: MetalCheckBoxUI inherits from MetalRadioButtonUI instead
     // of BasicCheckBoxUI because we want to pick up all the
     // painting changes made in MetalRadioButtonUI.
```

Comment:
Does it even apply here? Should it be instead?
```suggestion
    // NOTE: WindowsCheckBoxUI inherits from WindowsRadioButtonUI instead
    // of BasicCheckBoxUI because we want to pick up all the
    // painting changes made in WindowsRadioButtonUI.
```